Homework13/Tests/WebTableTests.cs

```diff
@@ -0,0 +1,138 @@
+﻿using Homework13.Data.Constants;
+using Homework13.PageObjects;
+using NUnit.Framework;
+
+namespace Homework13.Tests
+{
+    public class WebTableTests : BaseTest
+    {
+        const string firstName = "Rita";
+        const string lastName = "Bunets";
+        const string email = "email@example.com";
+        const string age = "26";
+        const string salary = "100000";
+        const string department = "QA";
+        const string entryValue = "Rita Bunets 26 email@example.com 100000 QA  ";
+
+        [SetUp]
+        public void SetUp()
+        {
+            GenericPages.BaseElementsPage.ClickOnElementsMenuItem(Elements.WebTables);
+        }
+
+        // Test1: Check validation of form inputs.
+        [Test]
+        public void CheckValidation()
+        {
+            const string validatedFormClass = "was-validated";
```

Comment:
Please incapsulate it in WebTablePage (add some method to check if form is validated)

Reply:
fixed

Reply:
fixed